packages/core/src/lib/notify.svelte

```diff
@@ -0,0 +1,127 @@
+<svelte:options immutable />
+
+<script lang="ts">
+type Variants = 'danger' | 'warning' | 'success' | 'info';
+
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon } from '$lib';
+
+export let title = '';
+export let message = '';
+export let variant: Variants = 'info';
+export let progress = 1;
+export let exitable = false;
+
+$: isExitable = exitable;
+
+const dispatch = createEventDispatcher();
+
+const handleClose = (event: Event) => {
+  dispatch('close', event);
```

Comment:
There's no need to pass the event as the value - just call `dispatch('close')`.

Reply:
the linter didnt like dispatch('close'). Error: Expected 2-3 arguments, but got 1.ts(2554)

Reply:
Ah, I'd do `dispatch('close', {})` and define the dispatcher as:

```
const dispatch = createEventDispatcher<{
  close: {}
}>();
```